### ISSUE TITLE
How to connecting to redis with password

### ISSUE THREAD

**Issue by vominhtam** [chrisboulton/php-resque#321]
Hi all,

I have issues, I want to connect to redis server which has a password. but I don't know how to do it.

Please help me. Thank you so much

**Comment by danhunsaker** [comment]
Simplest way is to pass `"redis://ignored:{$pass}@{$host}:{$port}"` to `Resque::setBackend()` before attempting to use Resque for anything.

**Comment by vominhtam** [comment]
hi danhunsaker,

I have used it but, It has issuse :

fsockopen() expects parameter 2 to be long, string given

thank you so much.

**Comment by danhunsaker** [comment]
That shouldn't be happening if you're using `dev-master`...

**Comment by vominhtam** [comment]
I using v 1.2.* . I have found solution. I use: Resque::redis()->auth('PASSWORD'); i have push to queue:high into redis. but I don't know how to run command to implement queue.

Please help me.

**Comment by danhunsaker** [comment]
Let me rephrase.  Do not use any version other than `dev-master`.  `1.2` is several years old, and considerably behind `dev-master` in both features and bugfixes.  The next release is close, but not quite ready.

**Comment by vominhtam** [comment]
It is resolved thank you so much. ^^!
